### PR TITLE
RegEx in the check function was not picking up environments so I adde…

### PR DESCRIPTION
**Pull request by Nemesisesq**
…d a '.+' to capture whitespace between double forward slashes and domaain

### CODE REVIEW

File: src/angular-environment.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ angular.module('environment', []).
 
 			angular.forEach(this.data.domains, function(v, k) {
 				angular.forEach(v, function(v) {
-					if (location.match(new RegExp("^http(s)?:\/\/" + v))) {
+					if (location.match(new RegExp("^http(s)?:\/\/.+" + v))) {
 						self.environment = k;
```
Comment (abey-alex) [inline src/angular-environment.js:91]: You should break here, once the environment is found.

**Comment by JoniJnm** [comment]
Convert wildcards to regex http://stackoverflow.com/a/13818704

**Comment by mcblum** [comment]
Just a quick note since I have an issue open as well - I tried this and it didn't fix whatever issue I'm seeing.

**Comment by juanpablob** [comment]
It's already fixed. I will release a new version tomorrow. Thank you!

**Pull request closed without merging**